src/t_cose_sign_verify.c

```diff
@@ -138,6 +139,7 @@ is_soft_verify_error(enum t_cose_err_t error)
 }
 
 
+#ifndef T_COSE_DISABLE_SIGN
```

Comment:
I think the main disable needed in this file is the code by the comment /* --- The array of COSE_Signatures --- */.  Make the call to process_cose_signatures() conditional (plus the Open/Close array). Return the error from there. The other functions can be conditional too, but even if they aren't -dead_strip will get rid of them.

Reply:
Updated.